beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -67,12 +71,29 @@ odoo.define('beesdoo_pos.screens', function (require) {
                     loaded.reject(err);
                 });
         },
+
+        auto_invoice : function() {
+        	var self = this;
+        	var customer = this.pos.get_client();
+        	var order = this.pos.get_order();
+        	if (customer && customer.is_company && !order.is_to_invoice()){
+            	self.click_invoice();
+        	}
+        	if (!customer && order.is_to_invoice()){
+        		self.click_invoice();
```

Comment:
There is still a problem of indentation here. I think (cf runbot) there is a mix of tabs and spaces.